### PR TITLE
[W5.11][W11-A1] wang zi

### PR DESCRIPTION
**Pull request by alexawangzi**
1) added a function to automatically list all entries after a new person is added with the add command 
2) added a sort command to sort entries based on alphabetical order of their name

### CODE REVIEW

**Review by alexawangzi — COMMENTED** [review]
I wrote the commit message for eaf073f wrongly. There is no explicit listAfterAdd command. It should be listAfterAdd function, which perfroms the listing automatically after the addition

**Review by ChengSashankh — COMMENTED** [review]
Your methods are well written. Just be aware of the coding standards for spacing, and be sure to avoid wildcard imports. You can do this by changing settings in your IDE if this is happening by default.

**Comment by wzneo** [comment]
Well done. It would be good to have comments describing the various functions.
I am not sure whether it would solve the automatic check, but updating your JUnit tests may solve the problem
Also, it might be better to submit a clean PR request for separate enhancements.

File: src/seedu/addressbook/commands/AddCommand.java [hunk 1]
```diff
@@ -67,10 +68,20 @@ public ReadOnlyPerson getPerson() {
     public CommandResult execute() {
         try {
             addressBook.addPerson(toAdd);
-            return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+            return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd),listAfterAdd());
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/AddCommand.java:71]: need space after `,` before `listAfterAdd`

File: src/seedu/addressbook/commands/AddCommand.java [hunk 2]
```diff
@@ -67,10 +68,20 @@ public ReadOnlyPerson getPerson() {
...
         } catch (UniquePersonList.DuplicatePersonException dpe) {
             return new CommandResult(MESSAGE_DUPLICATE_PERSON);
         }
     }
 
+
+
+    public List listAfterAdd(){
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/AddCommand.java:79]: Need header comments here.

Comment (okkhoy) [inline src/seedu/addressbook/commands/AddCommand.java:79]: Need space between `) {`

File: src/seedu/addressbook/parser/Parser.java [hunk 3]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (okkhoy) [inline src/seedu/addressbook/parser/Parser.java:14]: Don't import using *; import the necessary classes explicitly.

**Review by okkhoy — COMMENTED** [review]
Please check why your tests are failing

**Comment by okkhoy** [comment]
@alexawangzi 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**